Dockerfile

```diff
@@ -19,6 +19,7 @@ LABEL maintainer="team@cere.network"
 LABEL description="This is the optimization to create a small image."
 ARG PROFILE=release
 COPY --from=builder /cerenetwork/target/$PROFILE/cere /usr/local/bin
+COPY --from=builder /cerenetwork/target/release/wbuild/node-runtime /home/cere/wasm-file
```

Comment:
Rename `wasm-file` to `node-runtime-artifacts`

Reply:
Updated